content/plotly/concepts/express/terms/treemap/treemap.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: '.treemap()'
+Description: 'Returns a visualization of hierarchical data using nested rectangles.'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Plotly'
+  - 'Data'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.treemap()`** method in the express module from Plotly library allows the creation of charts in a hierarchy defined mainly by `labels` (`names` for `plotly.express treemap()`) and `parents` attributes. With `plotly.express treemap()` each row of the DataFrame is represented as a sector of the treemap. The number of elements in the list of `names` must be the same as the list of `parents`, and both must always be strings. If it is a dictionary, the number of elements in a given list taken as a value for the key `names` must be the same number of elements in a given list taken as a value for the key `parents`.
+
+## Syntax
+
+```pseudo
+plotly.express.treemap(
+    names = ["element1", "element2". "element3", "element4"]
+    parents = ["", "element1", "element1", "element1"] # Note: the empty element `""` also counts
+)
+
+update_traces()
+update_layout()
+show()
+```
+
+- `plotly.express.treemap()`: Usually used as `px`, and thus `px.treemap()`, this is the way to call the method in full.
+- `names`: The column which contains four rows - `element1`, `element2`, `element3`, `element4`.
+- `parents`: The column which containf four rows, each one corresponding to the rows of `names` - `""` for `element1`, `element5` for `element2`, `element6` for `element3`, `element7` for `element4`.
+- `update_traces()`: is used to update the treemap's root color. It can take arguments that allow to modify attributes such as colors, shapes, and styles.
+- `update_layout()`: this method updates the layout chart, allowing to tailor the appearence and visualization style, including margins, colors, and others.
+- `show()`: Is used to run and show the chart.
```

Comment:
```suggestion
- `plotly.express.treemap()`: Usually used as `px`, and thus `px.treemap()`, this is the way to call the method in full.
- `names`: The column which contains four rows - `element1`, `element2`, `element3`, `element4`.
- `parents`: The column which contains four rows, each one corresponding to the rows of `names` - `""` for `element1`, `element5` for `element2`, `element6` for `element3`, `element7` for `element4`.
- `update_traces()`: is used to update the treemap's root color. It can take arguments that allow to modify attributes such as colors, shapes, and styles.
- `update_layout()`: this method updates the layout chart, allowing to tailor the appearence and visualization style, including margins, colors, and others.
- `show()`: Is used to run and show the chart.
```